lib/legato/list_parameter.rb

```diff
@@ -18,6 +18,11 @@ def <<(element)
       self
     end
 
+    def |(element)
```

Comment:
Could you simply change `def <<(element)` on line 16 instead? Just adding `@elements.uniq!`. I think `|=` is something of a "monument to cleverness".

Alternatively, we could use a `Set` rather than an `Array` for the elements, and they would be unique. http://www.ruby-doc.org/stdlib-2.2.0/libdoc/set/rdoc/Set.html I'm not sure how far back in Ruby version Set goes.